GOVERNANCE.md

```diff
@@ -1,131 +1,187 @@
+OpenRefine is a meritocratic, consensus-based community project based on the [Apache Decision Making process](https://community.apache.org/committers/decisionMaking.html).
 
-All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
+In so far that is possilbe, decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion list](https://forum.openrefine.org/) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No major decisions about the project’s direction, bug fixes, or features should be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
 
-Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process. This document describes how that participation takes place.
+All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
 
 ### Users
 Users are community members who have a need for the project. Through their usage, they give the project a purpose. Users are encouraged to participate in the project life by providing feedback on how their needs are satisfied.
 
 Users can help the project by:
 
 - Advertising and advocating for the project
-- Informing developers of strengths and weaknesses of the tool through the [user discussion list](https://forum.openrefine.org/) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)
+- Informing developers of the strengths and weaknesses of the tool through the [user discussion list](https://forum.openrefine.org/) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)
 - Providing moral support (a ‘thank you’ goes a long way)
-- Writing tutorials
+- Writing tutorials to contextualize OpenRefine for specific communities
 
 How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!
 
 ### Contributors
 Contributors are users getting involved in the project more closely. Contributions can take many forms:
 
 - Supporting new users via the [user discussion list](https://forum.openrefine.org/c/support/12)
-- Submitting patches to fix bugs or add features via pull requests
+- Submitting patches to fix bugs or add features via pull requests, comment on pull requests and help other contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8)
```

Comment:
The previous bullet about peer support seems like it belongs in the Users section, rather than Contributors. Also, a couple of grammatical fixes here:

```suggestion
- Submitting patches to fix bugs or add features via pull requests, commenting on pull requests and helping other contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8)
```